### PR TITLE
Adjust the format for better readability

### PR DESCRIPTION
**Pull request by kmvan**
Adjust the format for better readability

Before submitting your pull request, check whether your code adheres to PHPMailer
coding standards by running the following command:

`./vendor/bin/php-cs-fixer --diff --dry-run --verbose fix `

And committing eventual changes. It's important that this command uses the specific version of php-cs-fixer configured for PHPMailer, so run `composer install` within the PHPMailer folder to use the exact version it needs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,7 +127,7 @@ try {
     $mail->send();
     echo 'Message has been sent';
 } catch (Exception $e) {
-    echo 'Message could not be sent. Mailer Error: ', $mail->ErrorInfo;
+    echo "Message could not be sent. Mailer Error: {$mail->ErrorInfo}";
 }
```
Comment (Synchro) [inline README.md]: The other changes are fine, but this one is not; It misunderstands how `echo` works.

Comment (kmvan) [reply README.md]: All the same.
But I think `"This is {$string}"` is more conducive to reading than `"This is", $string`, is it?

Comment (Synchro) [reply README.md]: If readability was the *only* concern, maybe, but concatenation or interpolation add unnecessary overhead (minimal though it is) with echo, so it's good practice to encourage the comma form.

Comment (kmvan) [reply README.md]: Let me do a performance benchmark, just wait a second.

Comment (kmvan) [reply README.md]: - OS: Arch x64
- PHP: 7.3.3 nts

```
<?php

// echoWithComma.php
// The result: 112bytes and 0.16124796867371s

declare(strict_types = 1);

$count         = 100000;
$isItRight     = 'is it right?';
$startMemUsage = \memory_get_usage();
$startTime     = \microtime(true);

while ($count > 0) {
    echo '<div style=display:none>Synchro said comma is good, ', $isItRight, '</div>';
    --$count;
}

$endMenUsage = \memory_get_usage() - $startMemUsage;
$endTime     = \microtime(true)    - $startTime;

echo "<h1>${endMenUsage}bytes</h1>";
echo "<h1>${endTime}s</h1>";
```
And next:
```
<?php

// echoWithoutComma.php
// The result: 112bytes and 0.10986399650574s

declare(strict_types = 1);

$count         = 100000;
$isItRight     = 'is it right?';
$startMemUsage = \memory_get_usage();
$startTime     = \microtime(true);

while ($count > 0) {
    echo "<div style=display:none>Synchro said comma is good, {$isItRight}</div>";
    --$count;
}

$endMenUsage = \memory_get_usage() - $startMemUsage;
$endTime     = \microtime(true)    - $startTime;

echo "<h1>${endMenUsage}bytes</h1>";
echo "<h1>${endTime}s</h1>";
```

Just easy to get the results. The withoutComma scheme is 68.133569% faster than withComma scheme (multi benchmark).

Comment (Synchro) [reply README.md]: You are quite right - I get similar results. So much for conventional wisdom!

**Comment by Synchro** [comment]
Thanks

**Comment by kmvan** [comment]
> Thanks

My honor

**Pull request merged**